x/sysproxy/README.md

```diff
@@ -0,0 +1,32 @@
+# Platform support
+
+The table below summarizes system-wide support this package offers for various types of proxies on desktop platforms.
+
+| Proxy Type | Windows | MacOS | Linux |
+| --- | ----------- | ------ | ------ |
+| HTTP | Yes | Yes | Yes
+| HTTPS | Yes | Yes | Yes
+| SOCKS | No | Yes | Yes
```

Comment:
I think you do support Windows, but it's SOCKS4?

Reply:
I found this socks server package that implements sock4, 4a, and 5:

https://github.com/cybozu-go/usocksd?tab=readme-ov-file

I wanted to test socks proxy on all platforms and see what versions they support in reality. I have tested linux and mac with socks 5 and they work fine. I could not get windows working with socks5 and it only worked with sock4 server in my tests. I have not tested mac and linux against socks4 though...Maybe we can do this later as TODO item.

Reply:
I have updated doc.go to reflect this in the documentation language